app/cryptography/jwe_decryption.py

```diff
@@ -52,19 +48,31 @@ def decrypt(self, token):
         encoded_cipher_text = tokens[3]
         encoded_tag = tokens[4]
 
-        decrypted_key = self._decrypt_key(encrypted_key)
+        decrypted_key = self._decrypt_key(encrypted_key, kid)
         iv = self._base64_decode(encoded_iv)
         tag = self._base64_decode(encoded_tag)
         cipher_text = self._base64_decode(encoded_cipher_text)
 
         signed_token = self._decrypt_cipher_text(cipher_text, iv, decrypted_key, tag, jwe_protected_header)
         return signed_token
 
-    def _decrypt_key(self, encrypted_key):
+    def _decrypt_key(self, encrypted_key, kid):
+
+        private_key = serialization.load_pem_private_key(
+            self._secret_store.get_private_key_by_kid(kid).encode(),
+            password="digitaleq".encode(),
```

Comment:
Seeing as the password is public it looks to offer little value, go-launch-a-survey also doesn't support password protected private keys.